src/helpers/utils.ts

```diff
@@ -278,3 +278,15 @@ export const filterSensitiveInfoFromRepository = (repositoryUrl: string | undefi
     return repositoryUrl
   }
 }
+
+// Removes sensitive info from the given git remote url and normalizes the url prefix.
+// "git@github.com:" and "https://github.com/" prefixes will be normalized into "github.com/"
+export const filterAndFormatGitRemote = (rawRemote: string | undefined): string | undefined => {
```

Comment:
nit (not related to this PR directly): Since this function only formats Github URLs it should maybe convey that in its name (`filterAndFormatGithubRemote`), or should it be made generic to support any domain?

Reply:
I think it makes sense to rename it to `filterAndFormatGithubRemote` for now, I'll make that change. Thanks